woocommerce/woocommerce
# Another set of migrations from set-output to GITHUB_OUTPUT

### All Submissions:






### Changes proposed in this Pull Request:

`set-output` is deprecated and will be removed early next year. This includes another set of migrations from `set-output` such as in #35799

Also note that saving these files ran Prettier against them too so where the formatting was incorrect that has also been fixed.

Note: I'm introducing these in chunks to make the test effort easier and to limit the risk of introducing wide ranging changes all at once.





### How to test the changes in this Pull Request:

1. You will need a fork of WooCommerce with Actions enabled. See testing instructions in #35799 if you need help with this.

2. Merge this branch into trunk of your fork

3. Test the is-community script. Create an arbitrary PR into your fork under your account. This should not be marked as a community contribution, check the CI check worked correctly and there were no errors. To test the case for community contributors, ask a team mate to submit an arbitrary PR to your fork. It should be marked as community contribution as long as they don't have write access to your fork.

4. In your fork, check what the stable version of WooCommerce is (in the readme.txt). Create a release in releases that is a version less than stable. Check under actions you should be able to see a "post release" action has been run. It should have a skipped step because the version released is less than current stable. Now create a new release that is greater than the stable release version, this workflow should run all steps and none should be skipped. (for both cases obviously no errors should be shown).

5. For the cherry pick, I think its quite difficult to test. @jonathansadowski have you tested cherry pick in PRs before?



### Other information:




### FOR PR REVIEWER ONLY:
